lib/sql.js

```diff
@@ -1159,8 +1159,9 @@ SQLConnector.prototype._buildWhere = function(model, where) {
  * @returns {string} The ORDER BY clause
  */
 SQLConnector.prototype.buildOrderBy = function(model, order) {
+  var idName = this.idName(model);
```

Comment:
@qpresley Good point and I agree. The responsibility of this function is to build corresponding SQL clause for `ORDER BY`. It's not the right place to set default order.